src/enums/typeNameEnum.js

```diff
@@ -0,0 +1,4 @@
+export const typeNameEnum = Object.freeze({
+  Person: "Person",
+  Query: "Query",
```

Comment:
as mentioned before, please adhere to naming conventions for property names in objects, use **camelCase**